app/js/sagas/labOrdersSaga.js

```diff
@@ -162,3 +164,18 @@ export function* setEncounters() {
   yield takeEvery(SET_LAB_ORDERS, fetchEncounters);
   yield takeEvery(UPDATE_LAB_ORDER_WITH_ENCOUNTER, fetchAndSetTestResultEncounter);
 }
+
+function* updateOrders() {
```

Comment:
instead of re-fetching all the orders again would have liked to just use the previous order uuid to filter state and just change the status to "cancel" but I found out that the new order returns with a different uuid. We could still do that knowing fully well that it will update whenever a refresh happens. @mogoodrich do you have any thoughts on this ?

Reply:
Yes, that makes sense, but I'm fine with just doing a full refresh for now.